core/src/test/java/net/opentsdb/query/processor/rate/TestRateConfig.java

```diff
@@ -66,31 +65,35 @@ public void serdes() throws Exception {
     assertTrue(json.contains("\"counterMax\":" + Integer.MAX_VALUE));
     assertTrue(json.contains("\"dropResets\":true"));
     assertTrue(json.contains("\"deltaOnly\":true"));
+    assertTrue(json.contains("\"rateToCount\":true"));
     
     json = "{\"id\":\"rate\",\"type\":\"Rate\",\"counter\":true,"
         + "\"interval\":\"60s\",\"dropResets\":true,\"counterMax\":2147483647,"
-        + "\"deltaOnly\":true}";
+        + "\"deltaOnly\":true,\"rateToCount\":true}";
     options = JSON.parseToObject(json, RateConfig.class);
     assertTrue(options.isCounter());
     assertTrue(options.getDropResets());
     assertEquals("60s", options.getInterval());
     assertEquals(Integer.MAX_VALUE, options.getCounterMax());
     assertTrue(options.getDropResets());
     assertTrue(options.getDeltaOnly());
+    assertTrue(options.getRateToCount());
   }
   
   @Test
   public void build() throws Exception {
     final RateConfig options = (RateConfig) RateConfig.newBuilder()
         .setCounter(true)
         .setInterval("60s")
+        .setRateToCount(true)
```

Comment:
also have a test case with interval `auto` please? want to check how that looks like,

Reply:
auto isn't used.